### PR TITLE
[expo] Allow to render `<AppLoading />` without props.

### PR DESCRIPTION
**Pull request by avocadowastaken**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [None of AppLoader props is required](https://docs.expo.io/versions/latest/sdk/app-loading.html#props)
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by huhuanming — APPROVED** [review]
👍

**Comment by huhuanming** [comment]
Thanks for your contribution!

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@
 //                 Tim Wang <https://github.com/timwangdev>
 //                 Kamal Mahyuddin <https://github.com/kamal>
 //                 Naoufal El Yousfi <https://github.com/nelyousfi>
+//                 Umidbek Karimov <https://github.com/umidbekkarimov>
```
Comment (timwangdev) [inline types/react-native/index.d.ts]: You may changed wrong files :) It's `types/react-native/index.d.ts` not `expo`

**Review by timwangdev — CHANGES_REQUESTED** [review]
Wrong file

**Review by KonstantinKai — CHANGES_REQUESTED** [review]
@umidbekkarimov , could you please add this changes for previous versions (v23, v24)?

**Review by KonstantinKai — APPROVED** [review]
@umidbekkarimov thanks 👍

**Comment by alloy** [comment]
@timwangdev’s comment has been addressed, so with @KonstantinKai’s :+1: this seems mergeable.

**Comment by janaagaard75** [comment]
I can see that I was late too late the to party for my comment to make it in time before this was merged. I hope we can wait a couple of days before merging the next time, so that we all have time to take a look at it.

**Pull request merged**